package.json

```diff
@@ -25,16 +25,10 @@
     "clean": "rm -rf build-test",
     "build-test": "npm run clean; babel -s -d built-test test",
     "lint": "standard src/**/*.js; standard --parser babel-eslint test/**/*.js",
-    "test": "npm run build-test; NODE_ENV=test node built-test/test.js",
-    "cover": "npm run build-test; NODE_ENV=test istanbul cover -x built-test/test.js built-test/test.js"
+    "test": "NODE_ENV=test node test/test.js",
+    "cover": "NODE_ENV=test istanbul cover -x test/test.js test/test.js"
```

Comment:
`async/await` is supported in latest stable node. I didn't switch from `.then` to `async/await` in the main library file for backwards-compatibility reasons.